website_multi_theme/models/website_theme.py

```diff
@@ -36,25 +37,54 @@ class WebsiteTheme(models.Model):
     def _convert_assets(self):
         """Generate assets for converted themes"""
         Asset = self.env["website.theme.asset"]
-        for one in self.filtered("converted_theme_addon"):
-            # Get all views owned by the converted theme addon
-            refs = self.env["ir.model.data"].search([
-                ("module", "=", one.converted_theme_addon),
-                ("model", "=", "ir.ui.view"),
-            ])
+
+        common_refs = self.env["ir.model.data"]
```

Comment:
You can delete this, the search below will return an empty recordset on his own if no results are found.

Reply:
My explanation is here: https://github.com/OCA/website/pull/443#discussion_r181154724
You probably miss it. What do you think about it?